lib/game.ex

```diff
@@ -48,8 +48,8 @@ defmodule TicTacToe.Game do
   end
 
   def build_computer_computer_game do
-    player_1 = ComputerPlayer.build("Computer 1", "\e[36mx\e[0m")
-    player_2 = ComputerPlayer.build("Computer 2", "\e[33mo\e[0m")
+    player_1 = SlowComputerPlayer.build("Computer 1", "\e[36mx\e[0m")
```

Comment:
This feels a bit too low level (name, ansi codes) to exist in a high level module like `Game`. Adds more reasons for this module to change. Could we maybe extract this out into its own module?